src/lib/InstantSearch.js

```diff
@@ -205,7 +205,7 @@ To help you migrate, please refer to the migration guide: https://community.algo
     });
 
     // Init the widget directly if instantsearch has been already started
-    if (this.started) {
+    if (this.started && Boolean(widgets.length)) {
```

Comment:
out of curiosity:why not just `&& widgets.length`?

Reply:
I'm a type lover I prefer when it's explicit - but agree it's exactly the same.